Makefile

```diff
@@ -280,10 +280,12 @@ ifndef ignore-not-found
   ignore-not-found = false
 endif
 
-clean-manifests = (cd config/components/manager && $(KUSTOMIZE) edit set image controller=us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue:$(RELEASE_BRANCH))
-clean-kueueviz-manifests = (cd config/components/kueueviz && \
-  $(KUSTOMIZE) edit set image backend=us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueueviz-backend:$(RELEASE_BRANCH) && \
-  $(KUSTOMIZE) edit set image frontend=us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueueviz-frontend:$(RELEASE_BRANCH))
+clean-manifests = \
+	(cd config/components/manager && \
+		$(KUSTOMIZE) edit set image controller=$(STAGING_IMAGE_REGISTRY)/kueue/$(IMAGE_NAME):$(RELEASE_BRANCH)) && \
+	(cd config/components/kueueviz && \
+  		$(KUSTOMIZE) edit set image backend=$(STAGING_IMAGE_REGISTRY)/kueue/$(IMAGE_NAME_KUEUEVIZ_BACKEND):$(RELEASE_BRANCH) && \
+  		$(KUSTOMIZE) edit set image frontend=$(STAGING_IMAGE_REGISTRY)/kueue/$(IMAGE_NAME_KUEUEVIZ_FRONTEND):$(RELEASE_BRANCH))
```

Comment:
Drop the use of IMAGE_NAME, IMAGE_NAME_KUEUEVIZ_BACKEND and IMAGE_NAME_KUEUEVIZ_FRONTEND

Reply:
Done